packages/server/graphql/types/Task.ts

```diff
@@ -181,6 +182,31 @@ const Task: GraphQLObjectType = new GraphQLObjectType<any, GQLContext>({
             sendToSentry(error, {userId: accessUserId})
           }
           return data
+        } else if (integration.service === 'gitlab') {
+          const gitlabAuth = await dataLoader
+            .get('teamMemberIntegrationAuths')
+            .load({service: 'gitlab', teamId, userId: viewerId})
+          const accessToken = gitlabAuth?.accessToken
+          if (!accessToken) return null
+          const {gid} = integration
+          const query = `
+            query {
+              issue(id: "${gid}"){
+                __typename
+                id
+                iid
+                title
+                ...info
```

Comment:
You mentioned [here](https://github.com/ParabolInc/parabol/pull/6160#discussion_r829211225) that we should use `...info` here as it'll pass the requested fields from the client to GitLab. 

Did you mean that we should include `...info` in addition to the other fields? Otherwise, we won't have access to those fields. 

To test: remove the other fields, like `id` and `title`, add an issue to the Estimate phase, and see that the title doesn't show up in the sidebar

Reply:
The `...info` should be enough, could you do a little triage and see if those fields are being included in the info.fieldNodes? perhaps a fragment is missing in the request somewhere? might be worth comparing the fragment for github and the one for gitlab to determine the delta

Reply:
you know, i'm not sure i love that design decision either... perhaps it should return the whole thing instead of just the info object? My goal was to make it hard to include fields in addition to the `...info`, but it may just be a gotcha